src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -8978,6 +8995,1050 @@ False
+                                                         ~~~~
+BC41009: The tuple element name 'f' is ignored because a different name is specified by the target type '(c As Integer, d As Integer)'.
+        Dim x2 As (a As Short, b As Short) = DirectCast((e:=1, f:=2), (c As Integer, d As Integer))
+                                                               ~~~~
+BC41009: The tuple element name 'e' is ignored because a different name is specified by the target type '(c As Long, d As Long)'.
+        Dim x3 As (a As Integer, b As Integer) = DirectCast((e:=1, f:="qq"), (c As Long, d As Long))
+                                                             ~~~~
+BC41009: The tuple element name 'f' is ignored because a different name is specified by the target type '(c As Long, d As Long)'.
+        Dim x3 As (a As Integer, b As Integer) = DirectCast((e:=1, f:="qq"), (c As Long, d As Long))
+                                                                   ~~~~~~~
+</errors>)
+
+        End Sub
+
+        <Fact>
+        <WorkItem(11288, "https://github.com/dotnet/roslyn/issues/11288")>
+        Public Sub TupleConversion02()
+
+            Dim comp = CreateCompilationWithMscorlibAndVBRuntime(
+<compilation name="Tuples">
+    <file name="a.vb">
+Module C
+    Sub Main()
+        Dim x4 As (a As Integer, b As Integer) = DirectCast((1, Nothing, 2), (c As Long, d As Long))
+    End Sub
+End Module
+        <%= s_trivial2uple %><%= s_trivial3uple %>
+    </file>
+</compilation>)
+
+            comp.AssertTheseDiagnostics(
+<errors>
+BC30311: Value of type '(Integer, Object, Integer)' cannot be converted to '(c As Long, d As Long)'.
+        Dim x4 As (a As Integer, b As Integer) = DirectCast((1, Nothing, 2), (c As Long, d As Long))
+                                                            ~~~~~~~~~~~~~~~
+</errors>)
+
+        End Sub
+
+        <Fact>
+        Public Sub TupleConvertedType01()
+
+            Dim comp = CreateCompilationWithMscorlibAndVBRuntime(
+<compilation name="Tuples">
+    <file name="a.vb">
+Module C
+    Sub Main()
+        Dim x As (a As Short, b As String)? = (e:=1, f:="hello")
+    End Sub
+End Module
+
+        <%= s_trivial2uple %>
+    </file>
+</compilation>)
+
+            Dim model = comp.GetSemanticModel(comp.SyntaxTrees(0))
+            Dim nodes = comp.SyntaxTrees(0).GetCompilationUnitRoot().DescendantNodes()
+            Dim node = nodes.OfType(Of TupleExpressionSyntax)().Single()
+
+            Assert.Equal("(e:=1, f:=""hello"")", node.ToString())
+            Assert.Equal("(e As System.Int32, f As System.String)", model.GetTypeInfo(node).Type.ToTestDisplayString())
+            Assert.Equal("System.Nullable(Of (a As System.Int16, b As System.String))", model.GetTypeInfo(node).ConvertedType.ToTestDisplayString())
+            Assert.Equal(ConversionKind.WideningNullable, model.GetConversion(node).Kind) ' TODO REVIEW
+
```

Comment:
It is a literal conversion form (1, "hello") to (short, string)? 
The underlying conversion (1, "hello") to (short, string) is widening, and the target is tuple, so WideningTuple seems correct.